File: Level1/12.문자열내림차순으로배치하기/12.문자열내림차순으로배치하기.java
import java.lang.StringBuilder;
class Solution {
    public String solution(String s) {
        StringBuilder answerSB = new StringBuilder(s);
        for(int i = 0; i<answerSB.length(); i++){
            for(int j = i+1 ; j<answerSB.length() ; j++){
                if(answerSB.charAt(i)<answerSB.charAt(j)){
                    char temp = answerSB.charAt(i);
                    answerSB.setCharAt(i, answerSB.charAt(j));
                    answerSB.setCharAt(j, temp);
                }
            }
        }
        String answer =answerSB.toString();
        return answer;
    }
}

File: Level1/12.문자열내림차순으로배치하기/12.문자열내림차순으로배치하기.py
def solution(s):
    answer = ''
    for i in range(len(s)):
        for j in range(i+1, len(s)):
            if s[i]<s[j]:
                s = s[:i]+s[j]+s[i+1:j]+s[i]+s[j+1:]
    answer = s
    return answer